backend/src/openarchiefbeheer/destruction/api/serializers.py

```diff
@@ -124,7 +124,32 @@ def __init__(self, *args, **kwargs):
     def validate_assignees(
         self, assignees: list[DestructionListAssignee]
     ) -> list[DestructionListAssignee]:
+        current_assignee_pks = (
+            list(
+                self.instance.assignees.filter(role=ListRole.reviewer).values_list(
+                    "user__pk", flat=True
+                )
+            )
+            if self.instance
+            else []
+        )
         assignees_pks = [assignee["user"].pk for assignee in assignees]
+
+        if current_assignee_pks and current_assignee_pks != assignees_pks:
+            comment = str(self.initial_data.get("comment")).strip()
+
+            if not comment:
+                raise ValidationError(
+                    _("A comment should be provided when changing assignees.")
+                )
+
+            logevent.destruction_list_reassigned(
+                destruction_list=self.instance,
+                assignees=assignees,
+                comment=comment,
+                user=self.context["request"].user,
+            )
```

Comment:
I think that this is a bit abusing this serializer :thinking: The serializer doesn't have a comment field, but we are checking for it in the case assignees are passed. This means that it won't automatically show up in the API docs. 
Also, what happens in the `create` case rather than the update? 

Maybe we need a separate serializer for updating the assignees :thinking:

Reply:
Maybe, I patched up the Serializer for now but maybe we should separate updates into different code paths. But having an update for a little bit of list data and another one for everything else feels a little awkward and could required a bigger refactor. I'l leave this up to you to decide.

Reply:
FYI: the comment field is not in the serializer but it's not required, only if assignees are set.

Reply:
Yes I saw! I am still not convinced by it, but if you want we can do a backend refactor in another issue

Reply:
> Yes I saw! I am still not convinced by it, but if you want we can do a backend refactor in another issue

Good Idea, I don't want to delay this issue by it. But don't get me wrong I'm in favor of refactoring this? Can you create an issue with changes you propose?

Reply:
https://github.com/maykinmedia/open-archiefbeheer/issues/174

Reply:
https://github.com/maykinmedia/open-archiefbeheer/issues/174